test/oai/test_client.py

```diff
@@ -93,21 +89,25 @@ def test_chat_completion():
 def test_completion():
     config_list = config_list_openai_aoai(KEY_LOC)
     client = OpenAIWrapper(config_list=config_list)
-    response = client.create(prompt="1+1=", model="gpt-3.5-turbo-instruct")
+    # Azure can't have dot in model/deployment name
```

Comment:
I saw this workaround in the code and looked like side effect which could lead to unexpected behaviour and decided to take it out... My bad I didn'thighlight it explicitly.

Is there any good from this feature, is it used? I.e.  if I define a model in Azure with name gpt-35-turbo than if I have it defined with the name gpt-3.5-turbo in the config and it still works it is kind of suspicious.  And there're way more model names than just the 2 previously used in the workaround (e.e gpt-3.5-turbo-1106)

From my experience working with Azure models it was only when I looked under the hood of Autogen and tried to achieve the tests running I had this thought that there might be a reason to follow a convention naming deployments after OpenAI public names of models but with dots removed, until than I just followed my own convention.

I can return the substitution back, it is easy though I would suggest to keep the code without implicit side effects and have a model names being exact matches to avoid confusion/issues in operation

Reply:
We can discuss this in a separate issue.